content/applications/general/product_images.rst

```diff
@@ -116,8 +114,8 @@ Variants list view. Here is a step-by-step guide from the Inventory app.
       If you select a product that has one or more variants from the Products view, each variant
       matching the previous criteria will be processed.
 
-#. In the action menu, select the option *Get Pictures from Google Images* and validate by clicking
-   on *Get picture*.
+#. In the action menu, select `Get Pictures from Google Images` and validate by clicking on 
+   `Get picture`.
 
 #. You should see your images appearing in the next few minutes.
```

Comment:
@chevalierv I would say 'You should see your images appearing incrementally'
Instead of 'few minutes'